test/functional/wallet_bumpfee.py

```diff
@@ -272,19 +325,43 @@ def test_locked_wallet_fails(rbf_node, dest_address):
     rbf_node.walletlock()
     assert_raises_rpc_error(-13, "Please enter the wallet passphrase with walletpassphrase first.",
                             rbf_node.bumpfee, rbfid)
+    rbf_node.walletpassphrase(WALLET_PASSPHRASE, WALLET_PASSPHRASE_TIMEOUT)
 
-
-def spend_one_input(node, dest_address):
+def test_change_script_match(rbf_node, dest_address):
```

Comment:
I think this could do with a doc string explaining what this subtest is testing.